src/libertem/executor/inline.py

```diff
@@ -20,19 +17,13 @@ def run_job(self, job, cancel_id=None):
             yield result
 
     def run_tasks(self, tasks, cancel_id):
-        # The UDFRunner expects one of these environment variable to be set
```

Comment:
So the default backend is now handled in the `UDFRunner`, right?

Reply:
Actually, in `common.backend.get_device_class()` which returns 'cpu' if no environment variable is set. This is then used in UDFRunner.